newIDE/app/src/Export/Builds/BuildsList.js

```diff
@@ -9,74 +9,165 @@ import EmptyMessage from '../../UI/EmptyMessage';
 import PlaceholderLoader from '../../UI/PlaceholderLoader';
 import { type AuthenticatedUser } from '../../Profile/AuthenticatedUserContext';
 import Text from '../../UI/Text';
-import { ColumnStackLayout } from '../../UI/Layout';
+import { ColumnStackLayout, ResponsiveLineStackLayout } from '../../UI/Layout';
 import { BuildCard } from './BuildCard';
 import PlaceholderError from '../../UI/PlaceholderError';
+import SelectField from '../../UI/SelectField';
+import SelectOption from '../../UI/SelectOption';
+import { type BuildType } from '../../Utils/GDevelopServices/Build';
 
+const styles = {
+  emptyMessageStyle: { textAlign: 'left' },
+};
 type Props = {|
   builds: ?Array<Build>,
   authenticatedUser: AuthenticatedUser,
   error: ?Error,
   loadBuilds: () => void,
   game: Game,
   onGameUpdated?: Game => void,
+  onBuildUpdated: Build => void,
+  onBuildDeleted: Build => void,
 |};
 
+type BuildFilter = BuildType | 'all-build';
+
+const selectOptionsArray: Array<{
```

Comment:
Ubernitpicking: "of course it's an array of options for a select, the type almost tell that, but are these options representing?"
=> give it a name like "buildFilterOptions" so that the reader directly understand "ok this represents the options for the build filters (and of course it's an array, there is an S to optionS and the type helps me understand that)"